contracts/examples/contracts/cross-token/ERC20TokenMessenger.sol

```diff
@@ -101,7 +101,7 @@ contract ERC20TokenMessenger is ReentrancyGuard {
             wrapped: false
         });
 
-        return messenger.sendUserXnetMessage{value: msg.value}(crossMsg);
+        return messenger.sendContractXnetMessage{value: msg.value}(crossMsg);
```

Comment:
Not strictly related to this PR, but should `return` be used here if `sendContractXnetMessage` doesn't return anything?

Reply:
We will end up deprecating this contract as part of the GMP work. This was a sample contract that used the previous implementation of GMP, so we can tackle it then.